dashboard/src/utils/axiosInstance.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { LOCAL_BASE_URL } from "./apiConstants";
+import Cookies from "js-cookie";
+import { showSessionExpired } from "actions/toastActions";
+import store from "store/store";
+
+const { dispatch } = store;
+
+let token = Cookies.get("token") ? Cookies.get("token") : null;
+
+const axiosInstance = axios.create({
+  baseURL: LOCAL_BASE_URL,
+  responseType: "json",
+  headers: {
+    Authorization: `Bearer ${token}`,
+  },
+});
+
+axiosInstance.interceptors.request.use(async (req) => {
+  token = Cookies.get("token") ? Cookies.get("token") : null;
+  req.headers.Authorization = `Bearer ${token}`;
+
+  return req;
+});
```

Comment:
Once again, if `Cookies.get("token")` returns `undefined`, what should `token` (or `req.headers.Authorization`) be?  `null` seems like an odd choice, given that it will stringize as `'null'`.  It seems like, if there's no token value, then we should be omitting the `Authorization` property altogether.  No?